### PR TITLE
Remove old todos

### PR DESCRIPTION
**Pull request by TimBuckers**

### CODE REVIEW

File: march_hardware/include/march_hardware/Joint.h [hunk 1]
```diff
@@ -17,7 +17,6 @@ class Joint
   TemperatureGES temperatureGES;
 
 public:
-  // TODO(Tim) pass by reference or pointer instead of making copy
```
Comment (TimBuckers) [inline march_hardware/include/march_hardware/Joint.h:20]: We tested this, but it's not easy. For now we are not planning on doing this.

File: march_hardware/test/TestEncoder.cpp [hunk 2]
```diff
@@ -12,14 +12,13 @@ using ::testing::Return;
 using ::testing::AtLeast;
 using ::testing::AtMost;
 
-class TestJoint : public ::testing::Test
+class TestEncoder : public ::testing::Test
```
Comment (TimBuckers) [inline march_hardware/test/TestEncoder.cpp:15]: fixed wrong name of test

File: march_hardware/test/TestEncoder.cpp [hunk 3]
```diff
@@ -12,14 +12,13 @@ using ::testing::Return;
...
 {
 protected:
-  // TODO(Tim) make parameterized
```
Comment (TimBuckers) [inline march_hardware/test/TestEncoder.cpp:18]: not needed for a mocked object

File: march_hardware/test/TestJoint.cpp [hunk 4]
```diff
@@ -25,8 +25,6 @@ class JointDeathTest : public ::testing::Test
     const float temperature = 42;
 };
 
-// TODO(Tim) write joint tests (with mocking)
```
Comment (TimBuckers) [inline march_hardware/test/TestJoint.cpp:28]: We tested this, but it's not easy. For now we are not planning on doing this.

**Review by Ishadijcks — APPROVED** [review]

**Pull request merged**